src/dashboard/src/fpr/migrations/0031_python_3.py

```diff
@@ -8,6 +8,7 @@
 OLD_THUMBNAIL_CMD_UUID = "7c2b65c7-6cea-4f81-9f3b-53375efc5bee"
 NEW_THUMBNAIL_CMD_UUID = "95149bc4-0620-4c20-964c-1d6c34b9400e"
 THUMBNAIL_RULES = (
+    "3a19f9a3-c5d5-4934-9286-13b3ad6c24d3",
```

Comment:
What does this do?

Reply:
It's a fallback rule to create a "default" thumbnail when all the other thumbnail rules can't (for example creating a thumbnail for a `.txt` file). Since we're introducing a new Python 3 compatible command to create the default thumbnail we need to update the existing rule (its current UUID and implementation was introduced in https://github.com/artefactual/archivematica-fpr-admin/pull/50).